### PR TITLE
fixed PDFTron intialization bug when no full api is used

### PR DESCRIPTION
**Pull request by jhruby**
We use XOD files only and when we upgraded to 8.2 we have got following error in the consol
obrazek-20211202-123404
e.

This code change has fixed the issue.

### CODE REVIEW

**Comment by bollain** [comment]
Hello @jhruby,

Thanks for this PR. I am not able to reproduce this issue using XOD files, which you can test here:
https://www.pdftron.com/samples/web/samples/viewing/viewing-with-custom-server/

This sample is running 8.2 and loads XOD files. 

Can you provide repro steps for this issue? 

If you wish you can submit a ticket to our support system to track this better.
https://www.pdftron.com/form/request/

**Comment by jhruby** [comment]
// here is shorter version of our initilization

   WebViewer(
               {
                  licenseKey: "ourlicense",
                  path:  "ourpath",
                  disabledElements: [
                     "thumbMultiDelete",
                     "thumbExtract",
                     "thumbDelete",
                     "dropdown-item-toolbarGroup-FillAndSign",
                     "dropdown-item-toolbarGroup-Insert",
                     "dropdown-item-toolbarGroup-Edit",
                     "dropdown-item-toolbarGroup-Forms",
                     "toolbarGroup-FillAndSign",
                     "toolbarGroup-Insert",
                     "toolbarGroup-Edit",
                     "toolbarGroup-Forms",

                  ],
                  isAdminUser: true,
                  annotationUser: "⠀"
               },
               viewer.current,
            ).then((instance) => {
               const {docViewer} = instance;
              const {Feature} = instance.UI;
             
// probably this
             instance.UI.enableFeatures([Feature.Copy]);

       
});

// here is Feature.Copy intialization
obrazek


// here is hotkeysManager.js where this was failing
obrazek

The hotkeysManager has not been initialized at time where we were calling enableFeatures([Feature.Copy]);  I have moved the initialization prior defineWebViewerInstanceUIAPIs. To make sure that enableFeatures call will not fail.

File: src/index.js [hunk 1]
```diff
@@ -154,25 +155,28 @@ if (window.CanvasRenderingContext2D) {
...
   defineWebViewerInstanceUIAPIs(store);
 
+
   fullAPIReady.then(() => loadConfig()).then(() => {
     if (preloadWorker) {
       initTransports();
     }
 
-    const { addEventHandlers, removeEventHandlers } = eventHandler(store);
-
```
Comment (bollain) [inline src/index.js:166]: What is the rationale for moving this down?

**Review by bollain — CHANGES_REQUESTED** [review]
Im able to reproduce this issue, and the fix addresses it. I tested the code and followed the logic and I see no other side effects from moving the initialization of the hotkeys manager.  However, there is one question of some code being moved around.

**Comment by bollain** [comment]
Hi @jhruby, I am closing this PR as we fixed this issue internally. You can see the fix here:
https://github.com/PDFTron/webviewer-ui/commit/56ec432655cc1c3ef6b0b96fa9d6ae5395823981

Thanks for your PR we were able to identify this issue, we appreciate your assistance in this regard!

**Pull request closed without merging**